benchmark-ph1ex3/coupled/model1-noTF.i

```diff
@@ -0,0 +1,316 @@
+v = -1701.71 # [cm/s]
+
+[GlobalParams]
+  num_groups = 3
+  num_precursor_groups = 8
+  use_exp_form = false
+  group_fluxes = 'group1 group2 group3'
+  sss2_input = true
+  account_delayed = false
+  # power = 55.704e6 # 350/2/pi 10^6 W
+  power = 350e6 # 350/2/pi 10^6 W
```

Comment:
Good catch.
This is something that might need further exploration.
In order to obtain the right flux magnitude, the normal_factor should be divided by 2*pi (as the problem uses coord_type = RZ).
The power, however, considers the integration over the 2*pi already, is the actual total power.
At some point in the past, I had a better understanding of this. I left the comment just in case someone in the future wants to further diagnose what is going on here.